INTERNALS.md

```diff
@@ -23,75 +23,188 @@ Serves endpoints through 3 controllers:
 Controllers use a variety of models to perform, in a traditional MVC approach.
 
 
-### Watch process
+### Worker process
 
-This process is built from 4 key classes:
+This process (of which multiple instances can be run) executes a number of
+threads concurrently.
 
-- The `Watch` service regularly polls for subscriptions and creates
-  a `Receive` service for each;
-- The `Receive` service, for a given subscriptions, buffers events from a `Queue` and creates
-  `Deliver` services to send the to clients;
-- The `Deliver` service gracefully sends event batches over HTTP;
-- The `Queue` model abstracts out queue management with Redis, providing a
-  syncronous means to push and pop message from a queue.
+- A group of `ROUTEMASTER_WORKER_THREADS` threads runs the `main` job queue
+  which delivers batches.
+- A single thread executes non-delivery jobs from the `aux` queue (monitoring,
+  auto-dropping, promotion of scheduled jobs)
+- A number of ticker threads schedule auxilliary jobs.
 
-# Cron process
+Details on the services above:
 
-This process runs scheduled tasks, regularly calling the following services:
-
-- `Monitor` delivers metrics to metric adapters;
+- `Monitor` delivers gauge metrics to metric adapters;
 - `Autodrop` automatically removes the oldest messages from queues under low
```

Comment:
`web`, `watch`, and `monitor` are processes, not services (as in, service classes). I'll try to rephrase to clarify though!
(using the "job" terminology which is probably more familiar)